### DISCUSSION TITLE
cypress coverage to derive dependency data

### DISCUSSION THREAD

**Discussion by tmc** [nrwl/nx#23412] [Feature Requests]
Is it feasible to use cypress coverage data to derive the dependency graph between an e2e test and particular files within a tree? I see more sophisticated custom graph capabilities in 0.17 and am curious if folks have considered this.

**Comment by FrozenPandaz** [comment]
We wouldn't want to run `cypress` tests in order to figure out the dependencies between projects. Do you have a suggest implementation for this?

**Reply by tmc** [reply]
I moreso meant collect coverage data from a cypress/playwright run (via istanbul, likely) and then turn that into dependency data so you know which high level tests to run when underlying things change.
